README.md

```diff
@@ -78,3 +78,51 @@ be heard and the call hung up.
 To uninstall all files associated with the example issue the following command:
 
     make uninstall-example
+
+## Asterisk Channel Variables
+
+You can use the following Respoke session information to manage them inside the asterisk channel:
+
+- respoke\_session\_local
+- respoke\_session\_local\_type
+- respoke\_session\_local\_connection
+- respoke\_session\_remote
+- respoke\_session\_remote\_type
+- respoke\_session\_remote\_connection
+- respoke\_session\_remote\_appid
+- respoke\_session\_id
+
+Here it is a dialplan example in order to pass the "respoke\_session\_remote" inside an asterisk channel variable:
+
+    exten => your_respoke_endpoint,1,Answer()
+    same => n,NoOp(RESPOKE METADATA: ${respoke_session_remote})
+    same => n,Ringing
+    same => n,Wait(8)
+    same => n,Playback(welcome)
+    same => n,SayAlpha(${respoke_session_remote})
+    same => n,Dial(SIP/300)
+    same => n,Hangup()
+
+In the above example, the "respoke\_session\_remote" information is played back to the caller using the Asterisk application SayAlpha.
+
+## AMI
+
+The following Event is available on to the Asterisk Manager Interface:
+
+**Event: respoke_session**
+
+##### Example:
+
+    Event: respoke_session
```

Comment:
We have modified the AMI event name according to your suggestion (RespokeSession).

Thank you for your suggestion.